tests/c_api_test/test_.py

```diff
@@ -234,7 +235,7 @@ def test_booster():
             ctypes.byref(out_len),
             result.ctypes.data_as(ctypes.POINTER(ctypes.c_double)))
         if i % 10 == 0:
-            print('%d iteration test AUC %f' % (i, result[0]))
+            print(f'{i} interation test AUC {result[0]:.6f}')
```

Comment:
```suggestion
            print(f'{i} iteration test AUC {result[0]:.6f}')
```

Reply:
I am so ashamed of this kind of errors, really sorry, will surely fix it. @StrikerRUS

Reply:
Oh, please don't be ashamed! We appreciate your help!